src/commands/leaderboard.js

```diff
@@ -58,7 +58,7 @@ export function createLeaderboardMessage(user, page) {
         .slice(page * PER_PAGE, (page + 1) * PER_PAGE);
 
     const embed = new MessageEmbed()
-        .setTitle("Leaderboard")
+        .setTitle("Leaderboard (Past 7 days)"
```

Comment:
eiku hetkonen mihin viiminen `)` katos :D
```suggestion
        .setTitle("Leaderboard (Past 7 days)")
```